website/static/js/phantomServer/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "phantomServer",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www"
```

Comment:
This seems incorrect. Shouldn't it be `node ./app.js`? Though this probably isn't necessary, given that we already have an invoke task.